src/ci_workflow/README.md

```diff
@@ -23,11 +23,12 @@ To use checks, nest them under `checks` in the manifest.
 
 The following checks are available.
 
-| name                                          | description                                                   |
-|-----------------------------------------------|---------------------------------------------------------------|
-| gradle:properties:version                     | Check version of the component.                               |
-| gradle:dependencies:opensearch.version        | Check dependency on the correct version of OpenSearch.        |
-| gradle:publish                                | Check that publishing to Maven local works, and publish.      |
+| name                                          | description                                                                       |
+|-----------------------------------------------|-----------------------------------------------------------------------------------|
+| gradle:properties:version                     | Check version of the component.                                                   |
+| gradle:dependencies:opensearch.version        | Check dependency on the correct version of OpenSearch in gradle properties.       |
+| gradle:publish                                | Check that publishing to Maven local works, and publish.                          |
+| package:version                               | Check dependency on the correct version of OpenSearch Dashboards in package.json. |
```

Comment:
This new check has a different naming structure than the previous version check, `gradle:properties:version`.  Can we align this new check to something like `npm:properties:version`?